deployment/ansible/roles/ckanext-datajson/tasks/main.yml

```diff
@@ -3,20 +3,16 @@
 - name: Install datajson dependencies
   command: "{{ ckan_virtualenv_path }}/bin/pip install jsonschema pyyaml lepl"
 
-- name: Install ckanext-datajson into ckan virtualenv
-  command: "{{ ckan_virtualenv_path }}/bin/pip install -e git+https://github.com/azavea/ckanext-datajson.git@{{ ckanext_datajson_version }}#egg=ckanext-datajson"
-  when: ckan_production
-
-- name: Symlink ckanext-datajson into ckan virtualenv
-  file: src={{ vagrant_ckanext_datajson_mountpoint }} dest={{ ckan_virtualenv_path }}/src/ckanext-datajson
-        state=link
+- name: Remove ckanext-datajson symlink from ckan virtualenv
+  file: dest={{ ckan_virtualenv_path }}/src/ckanext-datajson
+        state=absent
   when: not ckan_production
 
-- name: Run setup.py develop
-  command: "{{ ckan_virtualenv_path }}/bin/python setup.py develop
-            chdir={{ ckan_virtualenv_path }}/src/ckanext-datajson"
-  when: not ckan_production
+- name: Uninstall ckanext-datajson from ckan virtualenv if it is already there
+  command: "{{ ckan_virtualenv_path }}/bin/pip uninstall ckanext-datajson"
```

Comment:
The "remove" step could be taken out if we both destroy/recreate our VMs instead of reprovisioning one time.

The "uninstall" step I think ought to remain, because otherwise I don't think `pip` will actually install the extension if/when we change the version of `ckanext-datajson` and reprovision (because as far as `pip`  can tell, the versions are the same).

Reply:
Good point.  A simple version bump being applied rather than ignored seems like a reasonable expectation to support.  I think it's also reasonable to expect occasional to frequent destruction/recreation of development VMs.  That's certainly been my practice.  So I still vote for dropping the "remove" step.